### PR TITLE
Switch window: Adding Default Sync time 30 secs

### PR DESCRIPTION
**Pull request by ravirk91**
Bug 6997:Switch window action taking default sync time "-1"

### CODE REVIEW

File: Ginger/GingerCore/Actions/ActSwitchWindow.cs [hunk 1]
```diff
@@ -86,10 +86,24 @@ public Int32 WaitTime
             {
                 string val = GetInputParamValue("WaitTime");
                 Int32 intVal = -1;
+
                 if (String.IsNullOrEmpty(val))
+                {
```
Comment (JinendraGhodke) [inline Ginger/GingerCore/Actions/ActSwitchWindow.cs]: i think we can simplify it like below instead of too many if conditions. Test it by loading old xml to ensure we are not changing the value.
Please add also 2 unit test if value is not null or empty then wait time should be equal to the value specified
If value is  null or empty then it should be equal to 30

 Int32 intVal = 30;
if (!String.IsNullOrEmpty(val))
{
   Int32.TryParse(val, out intVal);
}

Comment (ravirk91) [reply Ginger/GingerCore/Actions/ActSwitchWindow.cs]: Pushed discussed changes and also added unit test
